File: 4_Java/Greater.java
// package 4_Java;

import java.util.Scanner;

public class Greater {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the two number: ");
        int a=sc.nextInt();
        int b=sc.nextInt();
        if(a>b){
            System.out.println(a+" is greater than "+b);
        }
        else{
            System.out.println(b+" is greater then "+a);
        }


        sc.close();
    }
}

File: 3_CPP/1_Conditionals/Greater.cpp
#include <iostream>
using namespace std;

int main(int argc, char const *argv[])
{
    cout << "Enter the two  numbers: ";
    int a, b;
    cin >> a >> b;
    if (a > b)
    {
        cout << "The greater number is " << a;
    }
    else
    {
        cout << "The greater number is " << b;
    }
    return 0;
}
